### DISCUSSION TITLE
Flatpak version

### DISCUSSION THREAD

**Discussion by A350** [LMMS/lmms#6838] [General]
Would it be possible to have with the new releases, version for Flatpak ?

**Comment by Veratil** [comment]
We don't maintain Flatpak, sorry.

**Comment by A350** [comment]
=(

**Comment by qnebra** [comment]
But if someone wants to do and maintain Flatpak version, do it

**Reply by RiedleroD** [reply]
I'm tempted, but I already got enough on my plate as-is
